### PR TITLE
Add one ./configure parameter --enable-amp-na-decode.

### PR DESCRIPTION
**Pull request by JasonTian518**
To add code for Ampere specific error, update Makefile.am
and configure.ac, add non-standard-ampere.c and
non-standard-ampere.h

Signed-off-by: Jason Tian <jason@os.amperecomputing.com>

### CODE REVIEW

File: non-standard-ampere.c [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include "ras-record.h"
+#include "ras-logger.h"
+#include "ras-report.h"
+#include "ras-non-standard-handler.h"
+#include "non-standard-ampere.h"
+
```
Comment (mchehab) [inline non-standard-ampere.c:20]: Where's the code? This patch seems incomplete...

**Comment by mchehab** [comment]
I'll wait for a new PR with everything needed for Ampere altogether.

**Comment by JasonTian518** [comment]
Hi Mauro，
I separate check-in to 3 upstream, the 1st one only add non-standard-ampere.c and non-standard-ampere.h, update script,  implementation code is ready and will be upstream in the other 2 upstream.
I don’t want to submit so many code in one upstream. I will combine it if you suggest. Thanks.

Jason Tian(田祥波)
Tel: +86-13651782166
[cid:image001.png@01D6FA51.857E4BF0]

**Comment by mchehab** [comment]
> I separate check-in to 3 upstream, the 1st one only add non-standard-ampere.c and non-standard-ampere.h, update script, implementation code is ready and will be upstream in the other 2 upstream. 
> I don’t want to submit so many code in one upstream. 

In this specific case, combining them is the right way. 

On other cases, a PR with multiple patches would also be OK, provided that each one is adding a new feature or addressing a bug. The rule is: one patch per logic change.

> I will combine it if you suggest. 

Thanks!

**Pull request closed without merging**